src/sentry/api/endpoints/user_emails.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
+
+from django.db import IntegrityError, transaction
+from django.utils.translation import ugettext as _
+from rest_framework import serializers, status
+from rest_framework.response import Response
+
+from sentry.api.bases.user import UserEndpoint
+from sentry.models import UserEmail
+
+
+class UserEmailSerializer(serializers.Serializer):
+    email = serializers.EmailField(required=True)
+    primary = serializers.BooleanField(required=False, default=True)
```

Comment:
should this actually default to true? seems better to make people explicitly set to primary

Reply:
Done.